pgjdbc/src/main/java/org/postgresql/translation/ru.po

```diff
@@ -771,13 +786,16 @@ msgid ""
 "Parameter of type {0} was registered, but call to get{1} (sqltype={2}) was "
 "made."
 msgstr ""
+"Параметр был зарегистрирован как {0}, а используется методget{1} (sqltype={2})."
 
 # key: postgresql.call.noreturntype
 #: org/postgresql/jdbc/PgCallableStatement.java:407
 msgid ""
 "A CallableStatement was declared, but no call to registerOutParameter(1, "
 "<some type>) was made."
 msgstr ""
+"This statement does not declare an OUT parameter.  Use '{' ?= call ... '}' "
+"to declare one."
```

Comment:
I can't read Russian but pretty sure this msgstr is English. Copy pasta?

Reply:
@sehrope  Are you reviewing the Russian translations now ? :)

Reply:
Ha! I know quite a few languages but most of them are of the programming variety ;-)

@vsli's comment about maintaining a translation changelog piqued my curiosity about how to programmaticaly generate the list of possible `msgid` values so was skimming the `.po` file and it caught my eye.

Reply:
This is certainly a copy-pasta type of thing...